src/gui/tray/usermodel.cpp

```diff
@@ -119,7 +115,7 @@ void User::slotBuildNotificationDisplay(const ActivityList &list)
             continue;
         }
         const auto message = AccountManager::instance()->accounts().count() == 1 ? "" : activity._accName;
```

Comment:
@claucambra I also had an idea to use `activity._message` for `Logger::instance()->postGuiLog(title, message);`. Right now, when we get multiple Talk message notifications with different text (**"Message1", "Message2", "Message4", etc.**), they will all result in just one notification to be displayed in the OS ("user has sent you a private message"). If we add an actual `activity._message`, instead of `const auto message = AccountManager::instance()->accounts().count() == 1 ? "" : activity._accName;`, then, we'll be seing the message in the OS popup. I think this is quite convenient and must work like that. You may want to consul @jancborchardt to get to know his opinion on that and whether we need to display multiple popups with **"Message1", "Message2", "Message4", etc.**